### DISCUSSION TITLE
How to disable MQT tele log messages in console?

### DISCUSSION THREAD

**Discussion by ottelo9** [arendst/Tasmota#15367] [Q&A] [answered]
I want to know how to disable this log messages in the web console:

```
08:30:12.375 MQT: tele/MT175/STATE = {"Time":"2022-04-14T08:30:12","Uptime":"1T11:54:02","UptimeSec":129242,"Heap":13,"SleepMode":"Dynamic","Sleep":50,"LoadAvg":19,"MqttCount":2,"Wifi":{"AP":1,"SSId":"FRITZ!Box","BSSId":"98:9B:CB:16:98:D7","Channel":1,"Mode":"11n","RSSI":60,"Signal":-70,"LinkCount":1,"Downtime":"0T00:00:05"}}
08:30:12.384 MQT: tele/MT175/SENSOR = {"Time":"2022-04-14T08:30:12","MT175":{"Total_in":10297.03,"Total_out":4.78,"Power_curr":317}}
```

I want to look for my print messages from my script. But this messages will the console.
I use SML and MQTT.

**Comment by sfromis** [comment, marked as answer]
If those messages appear more often than TelePeriod, you may have asked for data to transmitted "immediately" instead of only at TelePeriod.

**Reply by ottelo9** [reply]
No, they appear every TelePeriod. But I cant change this TelePeriod, because then I change the period of my MQTT send interval, too. I only want to disable this annoying log messages!

**Reply by sfromis** [reply]
No such micro-management features exist. You could publish your messages to a MQTT topic for your purpose, making it easy to use whatever tool for MQTT to follow what's published.

**Reply by ottelo9** [reply]
Ok thanks. So I could print my "debug" script messages via MQTT? How can I do this via script?

**Reply by sfromis** [reply]
Using the [Publish](https://tasmota.github.io/docs/Commands/#publish) command.
